### ISSUE TITLE
Scoring 0-4

### ISSUE THREAD

**Issue by A2Ramos** [w3c/silver#294]
Reviewing the WCAG 3 draft I wanted to commend the team for the refreshing approach towards Accessibility Standards

I wanted to provide some comments particularly in scoring.
It is great to see the 0 to 4 rating instead of the pass or fail approach.
Looking into the examples provided such as under text-alternatives, it would be great if there is more clarity on the section on the calculation of the percentage against issues.

Take for example 60% of all images have appropriate text alternatives.
What do we mean by "all images" does it refer to:
- Counting all image files on the website
- Counting all instances/occurrences where `<img>` tag as it appears on a the website

These are distinctly different as a company logo while it is one image file can be used on all pages using an `<img>` tag can be used on the same image multiple times.

Based on the examples, I understood it as looking at the `<img>` tag and not the image file.
If that is the case, another similar clarification would be is if we are counting the instance or occurrence.
Take for example a logo that appears on every page of a website of a 10 page website, the same `<img src="..logo" alt="company logo">` code is used.
Now consider another <img> tag, say `<img src="hero">` is used just on the homepage but this time it fails because there is no alt attribute.
Will the calculation for the rating be:

1/2 where 2 is counted from `<img src="hero">` and `<img src="..logo" alt="company logo">` are the 2 <img> instances on the website
or
1/11 where 11 is counted from `<img src="hero">` with `<img src="..logo" alt="company logo">` multiplied 10 times since it occurs on the website 10 times 

From my experience this has sometimes led to confusion for some due to misinterpreting the dividing number differently and having opinions on one approach being better than the other. An alternative approach to this is to count the issue against the number of pages of the website instead. Example if you have at least one image with no alt attribute on the page then it fails and this is divided against the total pages on the website. This too has its advantages and disadvantages but it tends to be easier to understand and explain.

While I am making this comment from a personal capacity, I am the Product Manager of a solution that includes automated accessibility checks so the information is very useful to know.

I look forward to further updates on WCAG 3.0

**Comment by lauracarlson** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by fstrr** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines \(WCAG\) 3.0](https://www.w3.org/TR/wcag-3.0/).  We believe that these issues are no longer relevant based on the changes made to the latest working draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
